packages/rs-dapi-client/src/mock.rs

```diff
@@ -257,32 +257,83 @@ impl Expectations {
 
 impl<R: Mockable> Mockable for ExecutionResponse<R> {
```

Comment:
We need to decided, either we keep mockable implementation in their modules, like address, or in mock, like execution types. I would say they should be in their modules otherwise the mock module will be a mess eventually.

Reply:
To be refactored in separate PR.